components/mixins/experience-event/experienceevent-card-actions.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-card-actions",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Card Actions",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Card action details such as card type as well as activation and lock status.",
+  "definitions": {
+    "card-actions": {
+      "properties": {
+        "xdm:card-actions": {
```

Comment:
hyphens are not supported in AEP. Please convert all hyphen fields to camel case - "cardActions" in this case.